index.html

```diff
@@ -0,0 +1,433 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>BINABOX</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <a href="#">Binabox <img src="#" alt="Picutre of logo" /></a>
```

Comment:
I've been trying to find the criteria, but so far I haven't been able to do it. I'm not sure, but it's possible that the criteria state that the logo should not lead anywhere for the main page. This means that you just need to remove the href attribute.